timemachine/maps/estimators.py

```diff
@@ -0,0 +1,56 @@
+"""Utilities for computing mapped works or mapped u_kn matrix, incorporating invertible map_fxns"""
+
+import numpy as np
+from pymbar.utils import kln_to_kn  # TODO: should this be "vendored"?
+
+__all__ = ["compute_mapped_reduced_work", "compute_mapped_u_kn"]
+
+
+def compute_mapped_reduced_work(src_samples, src_reduced_energy_fxn, dst_reduced_energy_fxn, map_fxn):
```

Comment:
Naming: (`samples`, `reduced_energy_fxn`) rather than (`xs`, `u_fxn`) (or (`confs`, `reduced_potential_fxn`) or similar), to anticipate the case where the samples / states / maps are augmented in some convenient way (e.g. `sample = (x, v)` to use a VelocityVerlet map, or `sample = (x, v, particle_selection)` to use a local MD map).